.github/workflows/fc-asciidoctor-pr.yaml

```diff
@@ -0,0 +1,38 @@
+name: fc-asciidoctor-pr
+on:
+  pull_request:
+    paths:
+      - utilities/fc-asciidoctor/**
+      - .github/workflows/fc-asciidoctor-pr.yaml
+jobs:
+  build:
+    env:
+      context: utilities/fc-asciidoctor
+      image_name: fc-asciidoctor
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v1
+      - name: Check and verify version.json
```

Comment:
this step can be released by the central action. see:
- https://github.com/redhat-cop/containers-quickstarts/blob/master/.github/workflows/build-s2i-python-kopf-publish.yaml#L18-L22

Reply:
Sorry, not making it easier for review.